test/common/components/RequiredLoginView.unit.spec.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import RequiredLoginView from '../../../src/js/common/components/RequiredLoginView.jsx';
+
+describe('<RequiredLoginView>', () => {
+  const anonymousUser = {
+    accountType: null,
+    dob: null,
+    email: null,
+    gender: null,
+    userFullName: {
+      first: null,
+      last: null,
+      middle: null,
+      suffix: null
+    }
+  };
+  const loa1User = {
+    accountType: 1,
+    dob: null,
+    email: 'fake@aol.com',
+    gender: null,
+    services: ['user-profile'],
+    userFullName: {
+      first: null,
+      last: null,
+      middle: null,
+      suffix: null
+    }
+  };
+
+  const loa3User = {
+    accountType: 3,
+    dob: '1984-07-17',
+    email: 'fake@aol.com',
+    gender: 'M',
+    services: ['facilities', 'hca', 'user-profile', 'edu-benefits'],
+    status: 'OK',
+    userFullName: {
+      first: 'WILLIAM',
+      last: 'RYAN',
+      middle: 'PETER',
+      suffix: null
+    }
+  };
+
+  const defaultProps = {
+    authRequired: 3,
+    serviceRequired: 'hca',
+    userProfile: loa1User,
+    loginUrl: 'http://fake-login-url'
+  };
+
+  function setup(props = {}) {
+    const mergedProps = Object.assign({}, defaultProps, props);
+    const tree = SkinDeep.shallowRender(
+      <RequiredLoginView {...mergedProps}>
+        <div>Test Child</div>
+      </RequiredLoginView>
+    );
+
+    const instance = tree.getMountedInstance();
+    instance.setState({ loading: false });
+    const vdom = tree.getRenderOutput();
+
+    return { tree, instance, mergedProps, vdom };
+  }
+
+  it('should render', () => {
+    const { vdom } = setup();
+    expect(vdom).to.not.be.undefined;
+  });
+
+  describe('LOA.current=1', () => {
```

Comment:
This isn't super important, but these could be a little more descriptive, e.g. 'user is LOA1' and 'app requires LOA3'